package.json

```diff
@@ -53,6 +53,7 @@
     "eslint-plugin-prefer-let": "^3.0.1",
     "eslint-plugin-promise": "^6.1.1",
     "postcss": "^8.4.31",
+    "prettier": "^3.2.5",
```

Comment:
Prettier is 8 MB in `node_modules`. I have about 30 projects on my machine, 8 MB is too much.

Just use Prettier from your IDE.